src/EPPlus/ExcelRangeBase.cs

```diff
@@ -1919,7 +1919,10 @@ public ExcelRangeBase Offset(int RowOffset, int ColumnOffset, int NumberOfRows,
         }
         /// <summary>
         /// Adds a new comment for the range.
-        /// If this range contains more than one cell, the top left comment is returned by the method.
+        /// If this range 
```

Comment:
Some new lines seems to be added here. Intended?

Reply:
Nope. Unintentional by an aggressive sorting through search results at one point. Fixing.